setup.cfg

```
@@ -1,11 +1,11 @@
 [metadata]
-name = qhub
+name = nebari
 author = Quansight
-url = https://github.com/quansight/qhub
-description = Management of QHub on Cloud Infrastructure
+url = https://github.com/quansight/nebari
+description = Management of Nebari on Cloud Infrastructure
 long_description = file: README.md
 long_description_content_type = text/markdown
-keywords = aws, gcp, do, azure, qhub
+keywords = aws, gcp, do, azure, nebari
 license = BSD 3-Clause License
 classifiers =
     Development Status :: 3 - Alpha
```

```suggestion
    Development Status :: 5 - Production/Stable
```
should be this or at least `Development Status :: 4 - Beta` as we are putting this into prod